mozart++/mpp_foundation/any.hpp

```diff
@@ -254,10 +392,10 @@ class mpp::any final {
         mpp::swap(m_data, val.m_data);
     }
 
-    any() = default;
+    constexpr any() = default;
 
     template <typename T>
-    /*implicit*/ any(const T &val) {
+    any(const T &val) {
         store(val);
     }
```

Comment:
Do not remove this `/*implicit*/`, to tell this from `mistakenly forgot to make it explicit`